typegate/tests/e2e/cli_deploy_test.ts

```diff
@@ -64,14 +64,13 @@ test("cli:deploy - automatic migrations", async (t) => {
     await shell(["git", "commit", "-m", "create migrations"]);
   });
 
-  await t.should("run migrations with `meta deploy`", async () => {
-    await meta(
-      "deploy",
-      ...nodeConfigs,
-      "-f",
-      "runtimes/prisma/prisma.py",
-    );
-  });
+  // not in t.should because it creates a worker that will not be closed
```

Comment:
if the issue is because of the deno runtime, I believe we shall manage it like other runtime (multiple instantiation possible) but disallow in practice (e.g. future python wasi), keep a reversible (that can be unmounted) memory stack can save us future headache!

Reply:
The command creates a new version of the typegraph and so new versions of the runtimes.